Gordon360/Services/ProfileService.cs

```diff
@@ -119,6 +119,20 @@ public IEnumerable<AdvisorViewModel> GetAdvisors(string id)
             //Set a list to return not null object in array
             return resultList;
         }
+
+        /// <summary> Gets the clifton strengths of a particular user </summary>
+        /// <param name="id"> The id of the user for which to retrieve info </param>
+        /// <returns> Clifton strengths of the given user. </returns>
+        public Clifton_Strengths GetCliftonStrengths(int id)
+        {
+            var strengths = _unitOfWork.CliftonStrengthsRepository.GetById(id);
+            if(strengths == null)
+            {
+                throw new ResourceNotFoundException() { ExceptionMessage = "Clifton Strengths not found." };
+            }
+            return strengths;
```

Comment:
Rather than returning the exact row from the DB, the typical first step to data sanitization would be to use a ViewModel that extracts the relevant subset of the database Model's properties. This also ensures that the data we're returning won't change if someone updates our table.

In this case, I think you can probably create a ViewModel that just has a list of strengths 1-5 and return only that info. Perhaps there's a usecase for Date Submitted as well, but you would need to discuss that with Chris Carlson and how he foresees using this data. If it's only for display on profiles, I would guess that just returning the strengths themselves is sufficient. You don't need any identifying info since the Controller is being called for a specific username.